charts/dv-pod/values.yaml

```diff
@@ -306,6 +320,10 @@ charon:
     # This is active only if 'privateKey' and 'existingSecret.name' are NOT set.
     # The generated key will be stored in a secret (e.g., "{{ .Release.Name }}-dv-pod-enr-key") 
     # with data keys 'private-key' (for the hex key) and 'public-enr' (for the ENR string).
+    # 
+    # NEW: If a secret exists with only 'private-key' (no 'public-enr'), the ENR job will
+    # automatically generate the public ENR from the private key and update the secret.
+    # This allows users to provide only the private key without needing to generate the ENR manually.
```

Comment:
Considering this is an unreleased chart, we probably don't have to say "New:", and could just incorporate the behaviour as normal